ghost/core/core/server/services/settings/SettingsBREADService.js

```
@@ -46,19 +46,22 @@ class SettingsBREADService {
                 return `Verify email address`;
             },
             getText(url, type, email) {
+                const expiration_message = 'For your security, the link will expire in 24 hours time.';
```

ESLint

[error] Identifier 'expiration_message' is not in camel case.

(camelcase)